src/main/kotlin/io/pivotal/intellij/jasmine/JasmineRunProfileState.kt

```diff
@@ -35,6 +35,14 @@ class JasmineRunProfileState(
         commandLine.exePath = interpreter.interpreterSystemDependentPath
         runSettings.envData.configureCommandLine(commandLine, true)
 
+        // Add debug options if we're in debug mode
+        val isDebug = executor.id == DefaultDebugExecutor.EXECUTOR_ID
+        if (isDebug) {
+            // Add Node.js debug options - use a fixed port for easier debugging
+            commandLine.addParameter("--inspect-brk=9229")
```

Comment:
This is a really nice improvement. But I don't think the comment is accurate, and I don't think adding `=9229` makes any difference. With or without it, Node tries to use port 9229 and fails to start up debugger support if something is already listening on that port. So I think it would be better to remove the explicit port number and the part of the comment that describes it.

Reply:
will try without it